packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.scss

```diff
@@ -12,7 +12,6 @@ $detailsList-text-color: $listTextColor;
 
 .focusZone {
   display: inline-block;
-  vertical-align: top;
```

Comment:
this could have potential side effects on some browsers. I think it could be ok, @ThomasMichon  any concerns?

Reply:
Spoke to David on phone. This is certainly a browser bug, the heights of the content and the container are exactly same, but IE doesn't line them up correctly unless I get rid of this vertical align. I actually feel a little less bad about this because it's only changing the DetailsList styles. Tested on Chrome and works for the basic cases and the demo page.

I'll go ahead and merge this.